messagetype.go

```diff
@@ -49,22 +49,22 @@ const (
 // where applicable).
 func (mt MessageType) SupportsTransaction() bool {
 	switch mt {
-	case Invalid0MessageType:
```

Comment:
I was looking to see how we did similar things as `SupportsTransaction` but then I noticed we can simplify it and improve readability